assets/src/edit-story/components/sidebar/provider.js

```diff
@@ -89,7 +83,7 @@ function SidebarProvider({ children }) {
         <Sidebar ref={ref}>
           {hasSidebar && (
             <SidebarContent top={offset}>
-              {type === TYPE_COLORPICKER && <ColorPicker {...props} />}
+              {sidebarState.displayContent && sidebarState.displayContent()}
```

Comment:
Preferably destruct this above as `const { renderContent } = sidebarState;` and then just assume it to be defined as a function here with `{renderContent()}`.

If it's not a function, it should fail hard anyway! No need to show a sidebar with no content.